docs/UserGuide.md

```diff
@@ -354,7 +354,7 @@ Lets you add an internship entry to be tracked by CareerSync!
 
 * The fields `COMPANY_NAME`, `DESCRIPTION`, `CONTACT_NAME` and `ROLE` allow the use of any text, number and/or spaces
 * The field `STATUS` accepts only the following inputs: `to_apply`, `ongoing`, `rejected`, `accepted` (case-insensitive)
-* The field `CONTACT_EMAIL` accepts only the format of `<email>@<domain>.com`
+* The field `CONTACT_EMAIL` accepts only the format of `<email>@<domain>`
```

Comment:
Amendment made is just to remove the strict specification in the UG that only ".com" email domains can be used.
As our email field accepts other domains as well